cover_agent/UnitTestGenerator.py

```diff
@@ -160,9 +160,9 @@ def build_prompt(self):
 
         return prompt.build_prompt()
 
-    def generate_tests(self, LLM_model="gpt-4o", max_tokens=4096, dry_run=False):
+    def generate_tests(self, LLM_model="gpt-4o", API_base="http://localhost:11434", max_tokens=4096, dry_run=False):
```

Comment:
I don't believe the intent was to hardcode a base URL into the script. Correct me if I'm wrong.
```suggestion
    def generate_tests(self, LLM_model="gpt-4o", max_tokens=4096, dry_run=False):
```

Reply:
I see the problem here. I'm going to refactor on your branch to move the AICaller instantiation to happen at the init phase of this class. Glad you surfaced this issue.